benches/benches/bevy_ecs/change_detection.rs

```diff
@@ -0,0 +1,228 @@
+use bevy_ecs::{
+    component::Component,
+    entity::Entity,
+    prelude::{Added, Changed},
+    world::World,
+};
+use criterion::{black_box, criterion_group, criterion_main, Criterion};
+use rand::{prelude::SliceRandom, SeedableRng};
+use rand_chacha::ChaCha8Rng;
+
+criterion_group!(
+    benches,
+    all_added_detection,
+    all_changed_detection,
+    few_changed_detection,
+    none_changed_detection,
```

Comment:
should we have 
- `few_added_detection`
- `none_added_detection`

?

Reply:
I would prefer to have them, but won't block on it.

Reply:
Let's focus on merging that PR then, we'll improve on that later